device/rsvp/rsvpTspec.yaml

```diff
@@ -0,0 +1,42 @@
+components:
+  schemas:
+    Rsvp.Tspec:
+      description: >-
+        Configuration for RSVP-TE TSPEC object included in Path Messages. The usage of these parameters is defined in RFC2215.
+      type: object
+      properties:
+        token_bucket_rate:
+          description: >-
+            The rate of the traffic to be carried in this LSP in bytes per second.
+            This is part of the Token Bucket specification defined for a traffic flow defined in RFC2215.
+          type: number
+          format: float
+          default: 0         
+        token_bucket_size:
+          description: >-
+            The depth of the token bucket in bytes used to specify the Token Bucket characteristics of the traffic to be carried
+            in the LSP.This is part of the Token Bucket specification defined for a traffic flow defined in RFC2215.
```

Comment:
LSP.This -> spaces after "."

Reply:
again, does 0 have special meaning?  what is the min/max?

Reply:
The specification doesn't disallow any value.It is upto the Dut to honour or dishonour. Dut dishonours by sending a RESV Err.One specification of Cisco, Cisco IOS-XR : https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/mp_te_path_setup/configuration/xe-16-8/mp-te-path-setup-xe-16-8-book/mpls-traffic-engineering-automatic-bandwidth-adjustment-for-te-tunnels.pdf
The bandwidth argument is the bandwidth, in kilobits persecond, set for the MPLS TE tunnel. The range is from 1 to 4294967295. The default is 0.This is not totally according to spec.

Reply:
Note that the example above was provided to indicate that there is no well defined min and max for this field. For units , we have set it to bytes/sec as per specification. The example above uses a different unit ( kilobits/sec ) which we are not using since it is different from the specification defined unit for this field.